recipe/build.sh

```diff
@@ -3,7 +3,7 @@
 cp $BUILD_PREFIX/share/gnuconfig/config.* ./TclMagick/unix/config
 cp $BUILD_PREFIX/share/gnuconfig/config.* ./config
 
-./configure --prefix=$PREFIX --enable-shared --enable-symbol-prefix --disable-dependency-tracking
+./configure --prefix=$PREFIX --enable-shared --enable-symbol-prefix --disable-dependency-tracking --with-quantum-depth=16 --disable-static --with-magick-plus-plus=yes
```

Comment:
This would break ABI right?

Reply:
Looking into Debian's transition process, it seems [it does indeed break the ABI](https://bugs.debian.org/cgi-bin/%3Ca%20href=%22bugreport.cgi?bug=796310%22%3E796310%3C/a%3E).

According to that bug report, they ended up compiling with the additional `--enable-quantum-library-names` option and they rebuilt dependent packages.

It might also be worth checking Homebrew's handling of the transition about 9 years ago in https://github.com/Homebrew/legacy-homebrew/issues/46101 https://github.com/Homebrew/legacy-homebrew/pull/46248 https://github.com/Homebrew/legacy-homebrew/pull/47014

Reply:
Apologies, I've corrected the link in my comment. Just for reference: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=796310